### ISSUE TITLE
Manuell implementation of config property setters/getters results in exception on resolve

### ISSUE THREAD

**Issue by mdepaly** [5minds/addict-ioc#48]
```
class Service1 {
 
 set config(value) {
    this._config = value;
    console.log('Configured to:', this._config);
  }
 
  get config() {
    return this._config;
  }
 
}

container.register('Service1', Service1)
.configure({confValue1: 'test'});
```
 
 results in 
```
Error: The setter for the config property on type 'Service1' is missing.
    at DependencyInjectionContainer._configureInstance 
```

**Comment by 5ebastianMeier** [comment]
This is due to different treatment of manually implemented getter and setter functions like these:

```javascript
public get firstName(): string {
   return this._firstName;
}
```

and auto-properties like these:

```javascript
public firstName: string = undefined;
```

I'm on it.

**Comment by 5ebastianMeier** [comment]
@mdepaly the issue should be resolved in `addict-ioc@1.0.6`.
